README.md

```diff
@@ -140,15 +183,47 @@ would look like for such an application.
 
 Your description:
 ```
-Replace this with your answer
+A RESTful route tells the controller how to handle the Emperor's demands, and
+what information it needs to retrieve from the database to render on the page.
 ```
 Your routes:
 ```
 The ancestors have provided an example of one route; you do the other six!
 
+GET '/warriors'
+  * This is the index route, which displays all warriors
+
+GET '/warriors/new'
+  * This is the new route, which displays a form where you can enter information
+  about a new warrior.
+
 GET '/warriors/:id'
   * This is the show route, which finds a warrior by ID, and displays information about that warrior.
 
+GET '/warriors/:id/edit'
+  * This is the edit route, which finds a warrior by ID and displays a page that
+  includes a delete button, should your warrior fall in battle, or just displeases you,
+  and a form where you can update a specific warrior's information.
+
+POST '/warriors'
+  * This is the create route, assuming the form on the '/warriors/new' had an
+  action of '/warriors' and a method 'post'. It should include a method creating
+  a new warrior using params from the form, and redirect to '/warriors' so
+  you can see the new warrior listed.
+
+PUT '/warriors'
+  * This is the update route, assuming the form on the '/warriors/:id/edit' page
+  had an input element of type hidden, method 'put', action '/warriors'. It should
+  include a method updating a warrior using params from the form, and redirect
+  to '/warriors'.
+
+DELETE '/warriors'
+  * This is the destroy route, assuming the delete button on the edit page had an
+  input type of hidden, method 'delete', action '/warriors'. It should include a
+  method to delete the warrior referenced in the params[:id] and redirect to
+  '/warriors'.
```

Comment:
:aries:  
/me Raising shield and sword to show respect for those who have fought bravely and well (you) and those who have fallen (no names).